src/figuresClasses.ts

```diff
@@ -1,11 +1,91 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.color = color;
+    this.shape = 'triangle';
+
+    const sides: number[] = [a, b, c].sort(
+      (i: number, j: number): number => i - j,
+    );
+
+    if (sides[0] <= 0) {
+      throw new Error('Side size <= 0');
+    } else if (sides[2] >= sides[0] + sides[1]) {
+      throw new Error('Wrong side size');
```

Comment:
[CODE STYLE] - The error message should be more descriptive. Consider changing it to 'The sum of any two sides must be greater than the third side' to clearly explain the triangle inequality principle.